ocaml/xapi/xapi_host.ml

```diff
@@ -2629,14 +2636,31 @@ let get_sched_gran ~__context ~self =
 let emergency_disable_tls_verification ~__context =
   (* NB: the tls-verification state on this host will no longer agree with state.db *)
   Stunnel_client.set_verify_by_default false ;
-  Unixext.unlink_safe Xapi_globs.verify_certificates_path
+  Unixext.unlink_safe Xapi_globs.verify_certificates_path ;
+  try
+    (* we update the database on a best-effort basis because we
+       might not have a connection *)
+    let self = Helpers.get_localhost ~__context in
+    Db.Host.set_tls_verification_enabled ~__context ~self ~value:false
+  with e ->
+    error "Failed to update database after TLS verication was disabled: %s"
+      (Printexc.to_string e) ;
+    raise
+      Api_errors.(
+        Server_error
+          ( internal_error
+          , ["Failed to update database after TLS verification disabled"]
```

Comment:
I think the message must be reworded to reduce user confusion, the exit status won't be 0 (success) even though this is the expected path:
"TLS verification disabled successfully. Failed to contact the coordinator to update the database."